src/CSS.php

```diff
@@ -22,6 +23,11 @@ class CSS extends Minify
     protected $maxImportSize = 5;
 
     /**
+     * @var array Chain of files to be imported
+     */
+    protected static $importChain = array();
```

Comment:
I prefer not to use a static property for this. Static properties are shared among all instances of a class, and one instance's `$importChain` is not relevant to another instance.